public/js/guests/controllers/rsvpController.js

```diff
@@ -0,0 +1,37 @@
+define([
+    '../module',
+    '../namespace',
+], function (module, namespace) {
+    'use strict';
+
+    var name = namespace + ".RsvpController";
+
+    module.controller(name, ['$scope', '$window', namespace + '.GuestServices', function ($scope, $window, GuestServices) {
+        $scope.guest = {};
+        $scope.attending = {};
+        $scope.attempts = 0;
+
+        $scope.validate = function (rsvpCode) {
+           GuestServices.get({ rsvp : rsvpCode }, function(guest) {
```

Comment:
Yeah, a previous commit comment said the "you have already rsvp'd" case hasn't been handled. I was thinking some kind of message with a mailto as well.